kolibri/core/auth/api.py

```diff
@@ -407,7 +408,7 @@ def get_session_response(self, request):
 
         response = Response(session)
         if not request.COOKIES.get("visitor_id"):
-            visitor_id = str(uuid4())
+            visitor_id = str(uuid4().hex)
             response.set_cookie("visitor_id", visitor_id, expires=visitor_cookie_expiry)
         else:
```

Comment:
Could this whole chunk just be removed (410-418), and we just rely on the cookie being set for anon users above?

Reply:
(Only reason I can think of to keep it is to best handle the case where someone stayed logged in for a year, so that we'd continue to refresh it? Given our Django session timeout policies, may not be a likely scenario)